scripts/build/luaTable.py

```diff
@@ -18,7 +18,7 @@ def dumpStream(out, d, lvl=0):
     def w(s):
         out.write(s)
 
-    if isinstance(d, str):
+    if isinstance(d, basestring): # fmt: skip
```

Comment:
Same idea as before. A portable version would be:

At the top of the file:
```
try:
    # Python 2 backward compatibility
    string_types = (basestring,)
except NameError:
    string_types = (str,)
```

Then write each check like this: `isinstance(d, string_types)`

Reply:
Ok.